harvard-lts/etd-integration-tests
# Alma Monitor Int Test

**Alma monitor int test**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-257
# How should this be tested?
Call https://ltsds-cloud-dev-1.lib.harvard.edu:10610/alma_monitor_service and wait for a Load report to appear in the dts@hu.onmicrosoft.com group email or the DRS2 Support Dev.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? yes
